pkg/preflights/host-preflight.yaml

```diff
@@ -127,6 +127,21 @@ spec:
         collectorName: 'wildcard-check'
         hostnames:
           - '*'
+    - subnetAvailable:
+        collectorName: Pods Subnet
+        exclude: '{{ or (ne .GlobalCIDR.CIDR "")  (eq .PodCIDR.CIDR "") }}'
+        CIDRRangeAlloc: '{{ .PodCIDR.CIDR }}'
+        desiredCIDR: {{.PodCIDR.Size}}
+    - subnetAvailable:
+        collectorName: Services Subnet
+        exclude: '{{ or (ne .GlobalCIDR.CIDR "")  (eq .ServiceCIDR.CIDR "") }}'
+        CIDRRangeAlloc: '{{ .ServiceCIDR.CIDR }}'
+        desiredCIDR: {{.ServiceCIDR.Size}}
+    - subnetAvailable:
+        collectorName: Subnet
+        exclude: '{{ eq .GlobalCIDR.CIDR "" }}'
+        CIDRRangeAlloc: '{{ .GlobalCIDR.CIDR }}'
+        desiredCIDR: {{.GlobalCIDR.Size}}
```

Comment:
Looks like you're offline, but I have a couple questions about these. Maybe we can link up tomorrow. I'll make some edits to the wording below for now.

Reply:
Talked with Salah and his comments cover my questions. I'm updating wording now.